config/user/default.yaml

```diff
@@ -1,5 +1,5 @@
 logdir:
-  root: ~/gflownet/logs
+  root: ./logs
```

Comment:
Just a comment, I think it's good enough, but I believe `.` is (at least in some settings) dependent on the directory from which you run the script, not relative to the file path; so if someone were to run from a different directory using `/complicated/path/to/their/gflownet/repo`, it might create `logs` directory relative to this (which might or might not be expected). In python at least, what I'd typically do is use the path relative to the file (e.g. `Path(__file__).parent`).

Again, I think this is fine, just FYI.

Reply:
That's the intended behaviour indeed. I think it's fine enough for a default value, which should be overwritten by the specific  user settings in practice. It was problematic before because it was creating a directory called `~`, which isn't fun...